user_guide_src/source/installation/upgrade_pagination.rst

```diff
@@ -0,0 +1,61 @@
+Upgrade Pagination
+##################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+==============
+
+- `Pagination Class Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/libraries/pagination.html>`_
+- :doc:`Pagination Documentation Codeigniter 4.X </libraries/pagination>`
+
+
+What has been changed
+=====================
+- In CI4 the pagination is a built-in method in the Model class. You have to change the views and also the controller in order to use the new pagination library.
```

Comment:
"In CI4 the pagination is a built-in method in the Model class." is not true.
Pagination in CI4 is also a library, but it is integrated with the Model. So if you use
CodeIgniter\Model, you use the built-in method in the Model class.